app.py

```diff
@@ -7,6 +7,7 @@
 from lib.user_repository import UserRepository
 from lib.user import User
 from lib.booking_repository import BookingRepository
+from functools import wraps
```

Comment:
Will we need to add functools into the pip file also so that it gets loaded within the environment?

Reply:
Actually no because turns out it's a default package which comes with the standard python library